### PR TITLE
Feature is GA now

### PR DESCRIPTION
**Pull request by spelluru**

### CODE REVIEW

**Comment by spelluru** [comment]
Feature is GA now. 
- Removed the preview sample folder
- Updated the GA sample script to use the latest commands.

**Comment by spelluru** [comment]
#sign-off

**Comment by spelluru** [comment]
@erickson-doug - please review and merge the changes when you get a chance. Thanks!!

File: event-grid/subscribe-to-blob-storage/subscribe-to-blob-storage.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+# You must have the latest version of the Azure CLI
+
 # Provide a unique name for the Blob storage account.
 storageName=contosostorage
```
Comment (sptramer) [inline event-grid/subscribe-to-blob-storage/subscribe-to-blob-storage.sh]: Storage name needs to be unique yet it isn't formatted the same as other user-supplied values. Change to be consistent.

**Pull request merged**